compose/compose-file/compose-file-v1.md

```diff
@@ -0,0 +1,663 @@
+
+    web:
+      links:
+       - db
+       - db:database
+       - redis
+
+Containers for the linked service will be reachable at a hostname identical to
+the alias, or the service name if no alias was specified.
+
+Links also express dependency between services in the same way as
+[depends_on](#dependson), so they determine the order of service startup.
+
+> **Note:** If you define both links and [networks](#networks), services with
+> links between them must share at least one network in common in order to
+> communicate.
+
+### log_driver
+
+> [Version 1 file format](compose-versioning#version-1) only. In version 2 and up, use
+> [logging](index.md#logging).
+
+Specify a log driver. The default is `json-file`.
+
+    log_driver: syslog
+
+### log_opt
+
+> [Version 1 file format](compose-versioning#version-1) only. In version 2 and up, use
+> [logging](index.md#logging).
+
+Specify logging options as key-value pairs. An example of `syslog` options:
+
+    log_opt:
+      syslog-address: "tcp://192.168.0.42:123"
+
+### net
+
+> [Version 1 file format](#version-1) only. In version 2 and up, use
+> [network_mode](index.md#networkmode) and [networks](index.md#networks).
+
+Network mode. Use the same values as the docker client `--net` parameter.
+The `container:...` form can take a service name instead of a container name or
+id.
+
+    net: "bridge"
+    net: "host"
+    net: "none"
+    net: "container:[service name or container name/id]"
+
+#### aliases
+
+Aliases (alternative hostnames) for this service on the network. Other containers on the same network can use either the service name or this alias to connect to one of the service's containers.
+
+Since `aliases` is network-scoped, the same service can have different aliases on different networks.
+
+> **Note**: A network-wide alias can be shared by multiple containers, and even by multiple services. If it is, then exactly which container the name will resolve to is not guaranteed.
+
+The general format is shown here.
+
+    services:
+      some-service:
+        networks:
+          some-network:
+            aliases:
+             - alias1
+             - alias3
+          other-network:
+            aliases:
+             - alias2
+
+In the example below, three services are provided (`web`, `worker`, and `db`), along with two networks (`new` and `legacy`). The `db` service is reachable at the hostname `db` or `database` on the `new` network, and at `db` or `mysql` on the `legacy` network.
+
+    version: '2'
+
+    services:
+      web:
+        build: ./web
+        networks:
+          - new
+
+      worker:
+        build: ./worker
+        networks:
+        - legacy
+
+      db:
+        image: mysql
+        networks:
+          new:
+            aliases:
+              - database
+          legacy:
+            aliases:
+              - mysql
+
+    networks:
+      new:
+      legacy:
+
+#### ipv4_address, ipv6_address
```

Comment:
this ipv4/6_address section is not in V1. It was also a section under `networks`. You can confirm by looking at `compose/config/config_schema_v1.json` in the `docker/compose` repo